test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+
+    @Test
+    public void isAnyNull() {
+        // no null
+        assertNoNull("123", 12);
+        assertNoNull("");
+
+        // only one null
+        assertHasNull("1232", null, 1009);
+        assertHasNull(10, null);
+        assertHasNull(null, 1223);
+
+        // multiple nulls
+        assertHasNull(20.0, null, null);
+        assertHasNull("Joks", null, null, null, null);
+        assertHasNull(null, null, null, "pop");
+
+
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
Do note that it is not necessary to abstract the `assertTrue` and `assertFalse` as separate methods (it was done previously to avoid having to type out Arrays.asList each time). However, it is also considered good practice to follow existing coding styles.